nerdlets/network-telemetry-overview/styles.scss

```diff
@@ -173,6 +173,9 @@ label {
 
 .select-account {
   margin: 20px;
+  span {
```

Comment:
What do you think of adding this spacing via `spacingType` on the `Icon` component?

It's a bit uglier in the JSX, but I think using the standard spacing values is worth it, especially if we're modeling the components for open source. I think `MEDIUM` or `LARGE` could work:

```js
<Icon type={Icon.TYPE.INTERFACE__ARROW__ARROW_LEFT} spacingType={[Icon.SPACING_TYPE.NONE, Icon.SPACING_TYPE.LARGE]} />
```

Reply:
Good idea!